sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosVectorIndexSpec.java

```diff
@@ -148,6 +152,38 @@ public CosmosVectorIndexSpec setIndexingSearchListSize(Integer indexingSearchLis
         return this;
     }
 
+    /**
+     * Gets the vector indexing shard key
+     *
+     * @return vectorIndexShardKey the list of string containing the shard keys used for partitioning the
+     * vector indexes. This applies to index types diskANN and quantizedFlat. The maximum allowed size for
+     * this array is 1 - that is, there is only one allowed path.
+     */
+    public List<String> getVectorIndexShardKey() {
```

Comment:
```suggestion
    public List<String> getVectorIndexShardKeys() {
```

This is a list of differnt keys, right?  Should we pluralize things (like comments, fields, etc)

Reply:
Yeah, that makes sense.